docs/pages/enroll-resources/database-access/database-access.mdx

```diff
@@ -1,49 +1,345 @@
+          href: "./enroll-aws-databases/aws-dynamodb/",
+          icon: "amazonDynamoDb",
+        },
+        {
+          name: "ElastiCache for Redis and Valkey",
+          href: "./enroll-aws-databases/redis-aws/",
+          icon: "amazonElastiCache",
+        },
+        {
+          name: "MemoryDB",
+          href: "./enroll-aws-databases/aws-memorydb/",
+          icon: "amazonMemoryDb",
+        },
+        {
+          name: "ElastiCache Serverless for Redis and Valkey",
+          href: "./enroll-aws-databases/elasticache-serverless/",
+          hidden: true,
+          icon: "amazonElastiCache",
+        },
+        {
+          name: "Keyspaces (Apache Cassandra)",
+          href: "./enroll-aws-databases/aws-cassandra-keyspaces/",
+          hidden: true,
+          icon: "amazonKeyspaces",
+        },
+        {
+          name: "OpenSearch",
+          href: "./enroll-aws-databases/aws-opensearch/",
+          hidden: true,
+          icon: "amazonOpenSearch",
+        },
+        {
+          name: "RDS Oracle",
+          href: "./enroll-aws-databases/rds-oracle/",
+          hidden: true,
+          icon: "amazonRds",
+        },
+        {
+          name: "RDS Proxy MySQL",
+          href: "./enroll-aws-databases/rds-proxy-mysql/",
+          hidden: true,
+          icon: "amazonRds",
+        },
+        {
+          name: "RDS Proxy for PostgreSQL",
+          href: "./enroll-aws-databases/rds-proxy-postgres/",
+          hidden: true,
+          icon: "amazonRds",
+        },
+        {
+          name: "RDS Proxy for Microsoft SQL Server",
+          href: "./enroll-aws-databases/sql-server-ad/",
+          hidden: true,
+          icon: "amazonRds",
+        },
+        {
+          name: "RDS and Aurora",
+          href: "./enroll-aws-databases/rds/",
+          hidden: true,
+          icon: "amazonRds",
+        },
+        {
+          name: "RDS for SQL Server",
+          href: "./enroll-aws-databases/rds-proxy-sqlserver/",
+          hidden: true,
+          icon: "amazonRds",
+        },
+        {
+          name: "Redshift Serverless",
+          href: "./enroll-aws-databases/redshift-serverless/",
+          hidden: true,
+          icon: "amazonRedshift",
+        },
+        {
+          name: "Redshift",
+          href: "./enroll-aws-databases/postgres-redshift/",
+          hidden: true,
+          icon: "amazonRedshift",
+        },
+      ]
+    },  
+    {
+      title: "Azure databases",
+      description: "Enroll and protect Azure managed databases.",
+      href: "./enroll-azure-databases/",
+      tags: [
+        {
+          name: "Azure SQL Server",
+          href: "./enroll-azure-databases/azure-sql-server-ad/",
+        },
+        {
+          name: "Azure Database for PostgreSQL",
+          href: "./enroll-azure-databases/azure-postgres-mysql/",
+        },
+        {
+          name: "Azure Redis",
+          href: "./enroll-azure-databases/azure-redis/",
+        },
+      ]
```

Comment:
Icons are adding a ton of polish, but Azure doesn't have them yet. Any plans to change that?